packages/wallets/particle/src/adapter.ts

```diff
@@ -0,0 +1,185 @@
+
+    private _connecting: boolean;
+    private _wallet: SolanaWallet | null;
+    private _publicKey: PublicKey | null;
+    private _config: ParticleAdapterConfig;
+    private _readyState: WalletReadyState =
+        typeof window === 'undefined' ? WalletReadyState.Unsupported : WalletReadyState.Loadable;
+
+    constructor(config: ParticleAdapterConfig) {
+        super();
+        this._connecting = false;
+        this._publicKey = null;
+        this._wallet = null;
+        this._config = config;
```

Comment:
No need  to assign this, or have the private property, since you're not using the config object at this time. The existence of a config param is to allow for future compatibility in case you add options later.